### PR TITLE
Init react redux

### PR DESCRIPTION
**Pull request by qwibbler**
Add website outline

### CODE REVIEW

**Review by elyor-doniyorov — COMMENTED** [review]
Hi @qwibbler, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that styling linter errors.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import { HashRouter as Router, Routes, Route } from 'react-router-dom';
+import Header from './components/header';
+import Books from './redux/books/books';
+import Categories from './redux/categories/categories';
+import './App.css';
+
```
Comment (BrianSammit) [inline src/App.js:6]: - [x] Currently, the app is not showing anything. Kindly make sure you fix this error. 

Capture


😄 🛠️

**Review by BrianSammit — CHANGES_REQUESTED** [review]
# CHANGES REQUIRED 🔴 🛑
Hello @qwibbler, great job you are doing so far in this project. 💯
TO HIGHLIGHT:

- Your project is professional 
- Your repo is professional 
- Well structured files 

However, there are a few changes to make to move to the next project 🛠️ . Keep working hard 💪 💻 

## Required Changes ♻️
See my inline comment to know what changes are needed. 👇 👀 

Please make these required changes and send it again for another review 👍 💪
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
# HAPPY CODING!!! 😄 

cat


**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

--------

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by D4nielJ — APPROVED** [review]
## Hello @qwibbler, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can always find me on slack!

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**